src/framework/components/collision/system.js

```diff
@@ -233,9 +233,9 @@ class CollisionSphereSystemImpl extends CollisionSystemImpl {
 // Capsule Collision System
 class CollisionCapsuleSystemImpl extends CollisionSystemImpl {
     createPhysicalShape(entity, data) {
-        const axis = (data.axis !== undefined) ? data.axis : 1;
-        const radius = data.radius || 0.5;
-        const height = Math.max((data.height || 2) - 2 * radius, 0);
+        const axis = data.axis ?? 1;
+        const radius = data.radius ?? 0.5;
```

Comment:
Doesn't this change functionality? I guess a zero radius doesn't make sense and previously it would turn it into 0.5:

![image](https://user-images.githubusercontent.com/5236548/218431540-62211930-32d6-43ba-85c6-ab7d48ebe851.png)

But then again, who would request a zero radius in the first place :thinking:

Reply:
I would argue that this was a bug before. While it's weird to request a 0 radius, I think the original intention was to catch `undefined`.